docs/content/stable/yugabyte-voyager/migrate/live-migrate.md

```diff
@@ -1363,3 +1396,4 @@ DROP USER ybvoyager;
 - For Oracle source databases, schema, table, and column names with more than 30 characters are not supported.
 - Sequences that are not associated with any column or are attached to columns of non-integer types are not supported for resuming value generation. These sequences must be manually resumed during the cutover phase.
 - For Oracle, only the values of identity columns on the migrating tables will be restored. The user will have to resume other sequences manually.
+
```

Comment:
add limitation to plain live-migration worfklow

For Postgres source, the following data types are not supported: 
"POINT", "LINE", "LSEG", "BOX", "PATH", "POLYGON", "CIRCLE", "GEOMETRY", "GEOGRAPHY", "BOX2D", "BOX3D", "TOPOGEOMETRY", "RASTER", "PG_LSN", "TXID_SNAPSHOT", "XML", "LO", "INT4MULTIRANGE", "INT8MULTIRANGE", "NUMMULTIRANGE", "TSMULTIRANGE", "TSTZMULTIRANGE", "DATEMULTIRANGE"